app/Services/GameService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Services;
+use App\Models\Wishlist;
+
+class GameService {
+    function getGamesById (int $id): Wishlist
+    {
+        $games = $this->getGames();
+
+        foreach ($games as $game) {
+            if ($game->id === $id) {
+                return $game;
+            }
+        }
+    }
+
+    function getGames(): array
+    {
+        $orderBy = '';
+        $direction = '';
+        $limit = 666;
+
+        return [
+        ];
+    }
+
+    function searchGamesByTitle (string $searchTitle)
```

Comment:
the idea for this function is to return a game, basically it's getGameById but by title.